src/mbed_os_tools/detect/platform_database.py

```diff
@@ -226,6 +226,13 @@
         u"1701": u"GD32_F307VG",
         u"1702": u"GD32_F450ZI",
         u"1703": u"GD32_E103VB",
+        u'1900': u'CY8CKIT_062_WIFI_BT',
+        u'1901': u'CY8CPROTO_062_4343W',
+        u'1902': u'CY8CKIT_062_BLE',
+        u'1903': u'CY8CPROTO_062_43012',
+        u'1904': u'CY8CPROTO_063_BLE',
+        u'1905': u'CY8CKIT_062_4343W',
+        u'1906': u'CYW943012P6EVB_01',
```

Comment:
Ids all accounted for! One little oddity: I noticed the following platforms have slightly different names in the database, just wondering if that was intentional:

- CY8CPROTO_062_43012 - CYPROTO-062-43012 (does not have the `8C` part after `CY`)
- CY8CPROTO_063_BLE - CYPROTO-063-BLE (does not have the `8C` part after `CY`)
- CY8CKIT_062_4343W - CY8CKIT-062-WiFi-BT-F (quite a different suffix)

Reply:
@bridadan: the names in this PR are correct, the update was sent to Mac today.

@maclobdell: can you confirm once the internal board database is updated?

All Cypress boards follow the consistent naming convention:

CY8CKIT - Pioneer Kit (with Arduino headers)
CY8CPROTO - Prototyping Kit (low-cost baseboard with connectivity module)
CY8CMOD - Connectivity module